lua/entities/ent_jack_claymore/init.lua

```diff
@@ -145,7 +145,10 @@ function ENT:Use( activator )
         local Tr = util.QuickTrace( activator:GetShootPos(), activator:GetAimVector() * 100, { self, activator } )
 
         if Tr.Hit and IsValid( Tr.Entity:GetPhysicsObject() ) then
-            if table.HasValue( PlantableMats, Tr.MatType ) then
+            local canConstrain = JID.CanConstrain( self, Tr.Entity ) or Tr.Entity:IsWorld()

```

Comment:
why would IsWorld be checked here when others dont?

Reply:
claymore welds itself to the world if it's planted ontop of grass, other mines just freeze themself, i'd like to keep this functionality